include/llvm/Transforms/IPO/StructFieldCacheAnalysis.h

```diff
@@ -0,0 +1,30 @@
+// llvm/Tranforms/IPO/StructFieldCacheAnalysis.h - Performs Cache-Aware Structure Analysis- C++ -*-===//
```

Comment:
This is not really a transform (since it doesn't change the IR), but nor is it an "analysis" pass in the llvm world (as in this doesn't provide an analysis for other transforms). I am fine with leaving it as such now, but may be comment this fact.